assets/app/view/game/map.rb

```diff
@@ -174,6 +174,22 @@ def render_map
           ]),
         ])
       end
+
+      def show_coords
+        default = false
+        stored = Lib::Storage['show_coords']
+        default = stored.nil? ? default : stored
```

Comment:
default = stored.nil? ? false : Lib::Storage['show_coords']

Reply:
toby, Why access Lib twice? ( the first time defines stored )

Reply:
`def show_coords
        stored = Lib::Storage['show_coords']
        default = stored.nil? ? false : stored`

Seems like maybe what you meant?

Reply:
yes thanks